features/00-allergies.feature

```diff
@@ -1,35 +1,46 @@
 @allergies-and-intolerances
```

Comment:
e.g. add `@use_case_clinical @allergies-and-intolerances` to include a "use case" tag.